src/main/kotlin/net/djvk/fireflyPlaidConnector2/transactions/PlaidFireflyTransaction.kt

```diff
@@ -0,0 +1,127 @@
+package net.djvk.fireflyPlaidConnector2.transactions
+
+import net.djvk.fireflyPlaidConnector2.api.firefly.models.TransactionTypeProperty
+import net.djvk.fireflyPlaidConnector2.api.plaid.models.Transaction
+import java.time.OffsetDateTime
+import java.time.ZoneId
+
+/**
+ * Represents a single non-transfer transaction, including both the Plaid and the Firefly representation of it.
+ */
+sealed interface PlaidFireflyTransaction {
```

Comment:
A description of the distinction between the subtypes and their intended lifecycle would be useful here.

Reply:
Better documentation added in https://github.com/nprzy/firefly-plaid-connector-2/commit/a1fc26cbe9641b94e500301e37b4d24adba14d54